src/linuxforhealth/csvtofhir/converter.py

```diff
@@ -136,17 +136,20 @@ def _convert_row_to_fhir(row: pd.Series) -> Tuple[Any, str, List[str]]:
         try:
             processing_exception: Optional[Exception] = None
             result: List[str] = []
-            log_message = (f"Converting row {row.rowNum} groupByKey={row.groupByKey} file_path={row.filePath} " +
-                           "resourceType={row.configResourceType}")
-            logger.debug(log_message)
+            logger.info((f"Converting row {row.rowNum} groupByKey={row.groupByKey} file_path={row.filePath} ") \
```

Comment:
I would remove groupByKey from this message.  I'd also remove resourceType (so just have file and row).

Reply:
I kept them because they were in the prior debug level message and I didn't want to provide less info than before. If we don't want these fields in the info message, then I think I should move those pieces into a debug-level message.